.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "wcwave_adaptors"]
-	path = wcwave_adaptors
-	url = https://github.com/tri-state-epscor/wcwave_adaptors.git
```

Comment:
Can you provide any details on how you included the wcwave_adaptors submodule? Also, why did you effectively remove this file?